job_templates/packaging_job.xml.em

```diff
@@ -35,6 +34,11 @@
             </a>
           </choices>
         </hudson.model.ChoiceParameterDefinition>
+        <hudson.model.StringParameterDefinition>
+          <name>CI_AMENT_BUILD_ARGS</name>
+          <description>Additional arguments passed to 'ament build'.</description>
+          <defaultValue>@(ament_build_args_default)</defaultValue>
+        </hudson.model.StringParameterDefinition>
```

Comment:
sounds good: https://github.com/ros2/ci/pull/84/commits/5c8a00e3ac4d81b02ac1e3c5258f67fee2219a43
This will result in a diff on all jobs config, and the position of the parameter will change in the UI (the build args will be just under the build type so much higher than before)